README.md

```diff
@@ -210,6 +211,10 @@ You can encrypt the `hsm_secret` content (which is used to derive the HD wallet'
 
 If you encrypt your `hsm_secret`, you will have to pass the `--encrypted-hsm` startup option to `lightningd`. Once your `hsm_secret` is encrypted, you __will not__ be able to access your funds without your password, so please beware with your password management. Also beware of not feeling too safe with an encrypted `hsm_secret`: unlike for `bitcoind` where the wallet encryption can restrict the usage of some RPC command, `lightningd` always need to access keys from the wallet which is thus __not locked__ (yet), even with an encrypted BIP32 master seed.
 
+### Data loss
+
+If you lose data (likely corrupted `lightningd.sqlite3`) about a channel __with `option_static_remotekey` enabled__, you can, after your peer unlateraly closed the channel, use `tools/hsmtool` with the `guesstoremote` command to (try to) recover your output from your peer's published commitment transaction.
+
```

Comment:
Not sure this belongs on the front-page. It's not like a corruption of `lightningd.sqlite3` is an everyday occurrence and this would be far better suited for a faq in the doc directory.

Reply:
Seemed to be frequently asked ... Hence the FAQ I guess ^^